Makefile

```diff
@@ -3,24 +3,27 @@ ROOT_DIR                        := $(patsubst %/,%,$(dir $(MAKEFILE_PATH)))
 
 VENDOR_SRC                      := $(ROOT_DIR)/vendors
 VENDOR_DIR                      := $(ROOT_DIR)/poetry/core/_vendor
+VENDOR_TXT                      := $(VENDOR_DIR)/vendor.txt
 POETRY_BIN                      ?= $(shell which poetry)
 
-
 .PHONY: vendor/lock
-vendor/lock:
-    # regenerate lock file
+vendor/lock: $(VENDOR_LOCK)
+	# regenerate lock file
 	@pushd $(VENDOR_SRC) && $(POETRY_BIN) lock
 
+.PHONY: vendor/sync
+vendor/sync:
 	# regenerate vendor.txt file (exported from lockfile)
 	@pushd $(VENDOR_SRC) && $(POETRY_BIN) export --without-hashes \
```

Comment:
Maybe we should pay attention to not catch too much output here, so maybe redirect `stderr` or somehow figure out how to take advantage of the `--output` flag, or maybe simply the `-q` flag.

Reply:
We can use the `-o` flag here

Reply:
I guess that would work, we would still need to exclude the lines that we do not need out of the file.